k8s-bench/tasks/configure-ingress/task.yaml

```diff
@@ -0,0 +1,6 @@
+goal: "Please create route for routing traffic from path '/app' to service 'web-service' in namespace 'ingress-test'."
+setup: "setup.sh"
```

Comment:
Idea: default these and use the presence of scripts to decide whether to run them